packages/aws-cdk/bin/cdk.ts

```diff
@@ -98,7 +98,7 @@ async function parseCommandLineArguments() {
       .option('strict', { type: 'boolean', desc: 'Do not filter out AWS::CDK::Metadata resources', default: false }))
     .option('fail', { type: 'boolean', desc: 'Fail with exit code 1 in case of diff', default: false })
     .command('metadata [STACK]', 'Returns all metadata associated with this stack')
-    .command('init [TEMPLATE]', 'Create a new, empty CDK project from a template. Invoked without TEMPLATE, the app template will be used.', yargs => yargs
+    .command('init [TEMPLATE]', 'Create a new, empty CDK project from a template.', yargs => yargs
```

Comment:
they do default to the app template but perhaps the messaging is still confusing.

What the help message is *trying* to say is that if you use `npx cdk init --language=typescript` or whatever language, you'll get the `app` template for that language.

Doing `cdk init` with no args basically gives you `cdk init --list`

Reply:
Hmm, indeed confusing. So maybe something like below would clarify and still be concise enough?

```suggestion
    .command('init [TEMPLATE]', 'Create a new, empty CDK project from a template. Invoked with language but without TEMPLATE, the app template will be used.', yargs => yargs
```